pkg/cli/cmd/certs.go

```diff
@@ -0,0 +1,162 @@
+package cmd
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/aserto-dev/topaz/pkg/cli/cc"
+	"github.com/aserto-dev/topaz/pkg/cli/certs"
+)
+
+type CertsCmd struct {
+	List     ListCertsCmd     `cmd:"" help:"list dev certs"`
+	Generate GenerateCertsCmd `cmd:"" help:"generate dev certs"`
+	Trust    TrustCertsCmd    `cmd:"" help:"trust dev certs"`
+}
+
+const (
+	DefaultCertsDir = "~/.config/topaz/certs"
+
+	gatewayCertFileName = "gateway"
+	grpcCertFileName    = "grpc"
+	certCommonName      = "topaz"
+)
+
+type ListCertsCmd struct {
+	CertsDir string `arg:"" required:"false" default:"~/.config/topaz/certs" help:"path to dev certs folder" `
```

Comment:
The UX feels asymmetric, generate requires named argument `--certs-dir` while `list` accepts an positional argument.

```
❯ ./dist/topaz_darwin_arm64/topaz certs generate --certs-dir="~/.config/topaz/certs2"

Generating gateway and grpc dev certs

Please wait .....
Done
❯ ./dist/topaz_darwin_arm64/topaz certs list --certs-dir="~/.config/topaz/certs2"
Usage: topaz certs list [<certs-dir>]

list dev certs

Arguments:
  [<certs-dir>]    path to dev certs folder

Flags:
  -h, --help        Show context-sensitive help.
  -N, --no-check    disable local container status check ($TOPAZ_NO_CHECK)

topaz: error: unknown flag --certs-dir
```

I would expect that:
`topaz certs list` lists the default directory
`topaz certs list --certs-dir="~/.config/topaz/certs2"` and we follow the same pattern and naming `generate` uses when specifying a different cert directory.